docs/api.rst

```diff
@@ -5020,17 +5020,17 @@ Exporting History Events
    :statuscode 409: No user is logged in or failure at event export.
    :statuscode 500: Internal rotki error
 
-.. http:patch:: /api/(version)/history/events/export
+.. http:put:: /api/(version)/history/events/export
 
-   Doing a PATCH on this endpoint with the given filter parameters will download a csv with all history events matching the filter. All arguments are optional. If no filter is used all the events will be downloaded.
+   Doing a PUT on this endpoint with the given filter parameters will download a csv with all history events matching the filter. All arguments are optional. If no filter is used all the events will be downloaded.
```

Comment:
is this change intentional? was it wrong?

Reply:
It's intentional, I'm pretty sure this was on develop before